packages/ui/src/2_molecules/Dropdown/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { DropdownCoords, DropdownMode } from './types';
+
```

Comment:
Rename the file to `Dropdown.utils.ts`. Make sure that every file in this PR is named according to https://sovryn.monday.com/docs/2733202752?blockId=275525d0-9985-4644-8023-61b8ead7d005 .